Mathlib/Data/ZMod/Defs.lean

```diff
@@ -6,6 +6,7 @@ Authors: Eric Rodriguez
 import Mathlib.Algebra.NeZero
 import Mathlib.Data.Nat.ModEq
 import Mathlib.Data.Fintype.Lattice
+import Mathlib.Topology.Order
```

Comment:
I don't think it is reasonable to import topology this early in the import hierarchy; can you put these results near wherever the discrete topology on `Nat` is established?

Reply:
Is it alright if I create a new file? More lemmas will be coming in shortly, regarding the equivalence of the induced and `embedProduct` topologies on `Units (ZMod n)`

Reply:
New file is fine!

Reply:
Don't forget to delete this import

Reply:
Done, thanks!

Reply:
How can I remove the ` Check all files imported (push) ` error?

Reply:
You should read https://github.com/leanprover-community/mathlib4/actions/runs/7286447002/job/19855177515?pr=9146 and add the line it's telling you to add to `Mathlib.lean`

Reply:
Thanks, done!